catalog/ocs-bundle.yaml

```diff
@@ -98,11 +98,6 @@ properties:
     group: ocs.openshift.io
     kind: OCSInitialization
     version: v1
-- type: olm.gvk
```

Comment:
@subhamkrai Can you drop this change also?

Reply:
reverted. Thanks!